samples/System.Numerics.Tensors.Samples/TensorSamples/WrapNativeMemory.NativeMemory.cs

```diff
@@ -9,23 +9,22 @@ namespace TensorSamples
 {
     partial class WrapNativeMemory
     {
-        public class NativeMemory<T> : OwnedMemory<T>
+        public class NativeMemory<T> : MemoryManager<T>
```

Comment:
Sample was out of date, needed cleanup. Not sure what all the appropriate changes here are, so I just did the minimal to get it to compile.

Reply:
CC. @GrabYourPitchforks, who indicated there may be a gist with the guidance

Reply:
Gist is here: https://gist.github.com/GrabYourPitchforks/8efb15abbd90bc5b128f64981766e834

Reply:
We had already fixed up `NativeMemory<T>` in the test project, so I just merged the code from there into the sample version.